src/Parallel/AlgorithmMetafunctions.hpp

```diff
@@ -18,6 +18,19 @@ template <typename Metavariables>
 class GlobalCache;
 /// \endcond
 
+/// The possible options for altering the current execution of the algorithm,
+/// used in the return type of iterable actions.
+enum AlgorithmExecution {
+  /// Proceed with the current set of iterable actions.
+  Continue,
+  /// Stop the execution of iterable actions, but allow entry methods
+  /// (communication) to restart the execution.
+  Pause,
+  /// Stop the execution of iterable actions and do  not wake up until after a
```

Comment:
remove extra space in between "do" and "not" and "wake up" -> "allow their execution"